bundles/ui/org.eclipse.smarthome.ui.classic/snippets/group_expanded.html

```diff
@@ -0,0 +1,2 @@
+%children%
+</ul></div>
```

Comment:
Yes, they are because of the way UI renderer works, it's the only ~convenient~ way I found to have a correct generated html.